### ISSUE TITLE
Problem to run "run_lstm_RGB.sh" on lisa-anna-public

### ISSUE THREAD

**Issue by haohao900618** [haohao900618/SCNN#1]
when I replicate the activity recognition experiments in the paper "Long-term Recurrent Convolutional Networks for Visual Recognition and Description", I download the repository from LisaAnne/lisa-caffe-public. I only change the path of UCF-101 frames to a pcpecific path, such as, RGB_frames = '~/frames' in sequence_input_layer.py file, but I got a strange promblem that the label of frames is decimals number only on test phase. I got the normal label on train phase.
some results when run "run_lstm_RGB.sh" :
I1210 09:15:40.268235 25677 solver.cpp:340] Test net output #42: reshape-label = 52.52
I1210 09:15:40.268242 25677 solver.cpp:340] Test net output #43: reshape-label = 46.68
I1210 09:15:40.268249 25677 solver.cpp:340] Test net output #44: reshape-label = 53.31
I1210 09:15:40.268257 25677 solver.cpp:340] Test net output #45: reshape-label = 52.52
I1210 09:15:40.268265 25677 solver.cpp:340] Test net output #46: reshape-label = 46.68
I1210 09:15:40.268272 25677 solver.cpp:340] Test net output #47: reshape-label = 53.31
I1210 09:15:40.268280 25677 solver.cpp:340] Test net output #48: reshape-label = 52.52
I1210 09:15:40.268287 25677 solver.cpp:340] Test net output #49: reshape-label = 46.68
I1210 09:15:42.790904 25677 solver.cpp:214] Iteration 0, loss = 4.61574
I1210 09:15:42.790948 25677 solver.cpp:229] Train net output #0: accuracy = 0.00260417
I1210 09:15:42.790963 25677 solver.cpp:229] Train net output #1: loss = 4.61574 (\* 1 = 4.61574 loss)
I1210 09:15:42.790972 25677 solver.cpp:229] Train net output #2: reshape-label = 89
I1210 09:15:42.790982 25677 solver.cpp:229] Train net output #3: reshape-label = 52
I1210 09:15:42.790988 25677 solver.cpp:229] Train net output #4: reshape-label = 74
I1210 09:15:42.790995 25677 solver.cpp:229] Train net output #5: reshape-label = 36
I1210 09:15:42.791002 25677 solver.cpp:229] Train net output #6: reshape-label = 21
I1210 09:15:42.791009 25677 solver.cpp:229] Train net output #7: reshape-label = 49
I1210 09:15:42.791016 25677 solver.cpp:229] Train net output #8: reshape-label = 77
I1210 09:15:42.791023 25677 solver.cpp:229] Train net output #9: reshape-label = 17
Is there anyone who has encountered the problem like me?
thank for your help!
